x-pack/plugins/uptime/public/components/fleet_package/certs_field.test.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { fireEvent, waitFor } from '@testing-library/react';
+import { render } from '../../lib/helper/rtl_helpers';
+import { CertsField, SSLRole, FieldKey, VerificationMode, FieldConfig } from './certs_field';
+
+// ensures that fields appropriately match to their label
+jest.mock('@elastic/eui/lib/services/accessibility/html_id_generator', () => ({
```

Comment:
i thought this module is exported by EUI, any idea why you need to mock this?

Reply:
There appears to be some sort of global mock that mocks it as a static string, which is unhelpful when you need each id to be unique. I didn't look too much into it. Happy to look into it more.